mail_composer_cc_bcc/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Email CC and BCC",
+    "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Social",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "maintainers": ["hailangvn2023"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "preloadable": True,
+    "depends": [
+        "mail",
+        "account",
```

Comment:
Q: why is this module depending on account?
I see it's only for overriding  a view for account invoice... can you please add to the ROADMAP that it should be moved out to a glue module?

Reply:
- this was to impact the `account.invoice.send` wizard as well in addition to the standard mail composer
- we took the assumption when working on v15 that most people have `account` installed in their db, but fair point, could be extracted to a more specific `mail_composer_cc_bcc_account`